pkg/prompt/prompt.go

```diff
@@ -60,29 +60,40 @@ func WakeUpPrompt(
 	ctxTimeout, cancel := context.WithTimeout(context.Background(), TIMEOUT)
 	defer cancel()
 
+	var isPrivateKeyBackup *bool = nil
+
 	for {
 		select {
 		case input := <-prompterApp.App().PromptInput:
 
 			var keepListening bool
-			var res interface{}
+			var res interface{} = nil
 			var err error
 
 			switch req.Action {
-			case walletapp.Delete, walletapp.Transfer, walletapp.Sign, walletapp.TradeRolls, walletapp.Backup:
+			case walletapp.Delete, walletapp.Transfer, walletapp.Sign, walletapp.TradeRolls, walletapp.Uncipher:
 				res, keepListening, err = handlePasswordPrompt(prompterApp, input, wallet)
 			case walletapp.NewPassword:
 				res, keepListening, err = handleNewPasswordPrompt(prompterApp, input)
 			case walletapp.Import:
 				res, keepListening, err = handleImportPrompt(prompterApp, input)
+			case walletapp.Backup:
+				if isPrivateKeyBackup == nil {
+					keepListening, err = handleBackupMethod(prompterApp, input)
+					isPrivateKeyBackup = &keepListening
+				} else {
+					res, keepListening, err = handlePasswordPrompt(prompterApp, input, wallet)
+				}
```

Comment:
can you add comments, I don't understand why you declare a bool and you only check if it is nil, not true nor false